package.json

```diff
@@ -2,13 +2,15 @@
   "name": "telegram-t",
   "version": "1.25.0",
   "description": "",
+  "homepage": "https://ekumenlabs.github.io/telegram-tt",
   "main": "index.js",
   "scripts": {
     "dev": "cross-env APP_ENV=development webpack serve --env mode=dev --env isDevServer --mode development --config ./webpack.config.js",
     "build": "webpack --mode production",
     "build:staging": "rm -rf dist/ && APP_ENV=staging npm run build && ./deploy/copy_to_dist.sh",
     "build:production": "npm i && rm -rf dist/ && APP_VERSION=$(npm run inc_version --silent) APP_ENV=production npm run build && ./deploy/copy_to_dist.sh",
     "deploy:production": "npm run build:production && git add -A && git commit -a -m '[Build]' --no-verify && git push",
+    "deploy:staging": "npm run build:staging && gh-pages -d dist --message '[EKUDEPLOY] Deployed to Github Pages'",
```

Comment:
Let's have this for `production`.

Reply:
Done in 6f4ef31293c0de939c0fbe4c10ff5992ad42a366